pkg/cmd/server/bootstrappolicy/web_console_role_test.go

```diff
@@ -0,0 +1,102 @@
+package bootstrappolicy
+
+import (
+	// standard lib
+
+	"strings"
+	"testing"
+	// kube stuff
+
+	"k8s.io/apimachinery/pkg/util/sets"
+	// openshift stuff
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+)
+
+// NOTE: If this test fails, talk to the web console team to decide if your
+// new role(s) should be visible to an end user in the web console.
+
+var rolesToHide = sets.NewString(
+	"cluster-admin",
+	"cluster-debugger",
+	"cluster-reader",
+	"cluster-status",
+	"registry-admin",
```

Comment:
To add to what Ben said, `rolesToShow` is used by the test so it must include all roles, not just the Kube ones.  I am considering a followup to make it so the Origin roles are not manually annotated, and instead use the same logic with `rolesToShow`.  I think this will make it easier in the long run to keep things in sync (only two lists to keep up to date).